hack/test-integration.sh

```diff
@@ -108,19 +109,14 @@ function exectest() {
 		result=$?
 	fi
 
-	tput cuu 1 # Move up one line
-	tput el		# Clear "running" line
+	os::text::clear_last_line
 
 	if [[ ${result} -eq 0 ]]; then
-		tput setaf 2 # green
-		echo "ok      $1"
-		tput sgr0		# reset
+		os::text::print_green "ok      $1"
 		exit 0
 	else
-		tput setaf 1 # red
-		echo "failed  $1"
-		tput sgr0		# reset
-		echo "${out}"
+		os::text::print_red_bold "failed  $1"
+		os::text::print_red "${out}"
```

Comment:
That's the current output of `test-cmd`. Figured it would be OK here, too. Can revert if you'd prefer.